README.md

```diff
@@ -9,49 +9,34 @@
 
 [OpenVoiceOS](https://openvoiceos.org/) is an open source platform for smart speakers and other voice-centric devices.
 
-[Mycroft](https://mycroft.ai) is a hackable, open source voice assistant by MycroftAI. OVOS-core is a
-backwards-compatible descendant of [Mycroft-core](https://github.com/MycroftAI/mycroft-core), the central component of
-Mycroft. It contains extensions and features not present upstream. All Mycroft Skills and Plugins should work normally
-with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
+[Mycroft](https://mycroft.ai) was a hackable, open source voice assistant by the now defunct MycroftAI. OpenVoiceOS continues that work and ovos-core (this repo) is the central component.
+
+All Mycroft Skills and Plugins should work normally with OVOS-core. Other Mycroft-based assistants are also believed, but not guaranteed, to be compatible.
 
 The biggest difference between OVOS-core and Mycroft-core is that OVOS-core is fully modular. Furthermore, common
 components have been repackaged as plugins. That means it isn't just a great assistant on its own, but also a pretty
 small library!
 
-Furthermore, it offers a number of cli bindings. The old Mycroft shell scripts still exist, and still work, but that
-stuff is now built into the Python program (docs to follow in the form of `--help`, because it's a lot.)
-
----
-
-**Installing OVOS-core** (NOTE: at this early stage, required system libs are presumed, and your distribution might be a
-question mark.)
-
-We suggest you do this in a virtualenv:
-
-`pip install ovos-core[mycroft]`
-
----
-
-As always, the OpenVoiceOS team thanks the following entities (in addition to MycroftAI) for making certain code and/or
-manpower resources available to us which may not have been compatible with our practices before:
-
-- NeonGecko
-- KDE
-- Blue Systems
-
 ## Table of Contents
 
-- [Running Mycroft](#running-mycroft)
+- [Installing OVOS](#installing-ovos)
 - [Skills](#skills)
 - [Getting Involved](#getting-involved)
 - [Links](#links)
 
+
+## Installing OVOS
+
+We strongly suggest you use the companion project [ovos-installer](https://github.com/OpenVoiceOS/ovos-installer) to install OVOS, several repositories are needed to get the full assistant running
```

Comment:
Maybe
If you have an existing system that you would like to install OVOS on, we strongly suggest...
and
Mention the buildroot and raspOVOS images for rpi

Reply:
I'm mostly using the installer on Raspberry Pi and this is what the trend indicates as well based on the telemetry.

On the last 60 installs _(since the DB got corrupted during my TrueNAS crash...)_.
```
     17 n/a
      1 Raspberry Pi 3 Model A Plus Rev 1.0
      1 Raspberry Pi 3 Model B Plus Rev 1.3
      6 Raspberry Pi 3 Model B Rev 1.2
      2 Raspberry Pi 4 Model B Rev 1.1
     10 Raspberry Pi 4 Model B Rev 1.2
      6 Raspberry Pi 4 Model B Rev 1.4
     13 Raspberry Pi 4 Model B Rev 1.5
      3 Raspberry Pi 5 Model B Rev 1.0
      1 Raspberry Pi Zero 2 W Rev 1.0
```

Reply:
Does it make sense to call out buildroot yet? Agreed with Jer's wording otherwise